app/os/main.cpp

```diff
@@ -322,6 +325,16 @@ namespace mcuboot {
 
 }	// namespace mcuboot
 
+namespace activities {
+
+	auto display_tag = leka::activity::DisplayTags(rfidkit, display::videokit);
+
+	inline const std::unordered_map<MagicCard, interface::Activity *> activities {{MagicCard::number_10, &display_tag}};
+
+}	// namespace activities
```

Comment:
pour faire comme les commandes :

```suggestion
namespace activity {

	namespace internal {
	
		auto display_tag = leka::activity::DisplayTags(rfidkit, display::videokit);
	
	}

	const std::unordered_map<MagicCard, interface::Activity *> list {{MagicCard::number_10, &display_tag},};

}	// namespace activities
```